probe-rs/src/vendor/st/sequences/stm32_armv7.rs

```diff
@@ -90,12 +105,11 @@ impl ArmDebugSequence for Stm32Armv7 {
         memory: &mut dyn ArmMemoryInterface,
         _core_type: CoreType,
     ) -> Result<(), ArmError> {
-        let mut cr = dbgmcu::Control::read(&mut *memory)?;
-        cr.enable_standby_debug(false);
```

Comment:
@inazarenko, I’m curious why these were no longer disabled in your recent changes. Could you clarify the reason?

Reply:
I don't explicitly change those bits because instead I restore the contents of the entire register to the value encountered and saved when attaching the debugger.